edbdeploy/project.py

```diff
@@ -1144,7 +1144,7 @@ def ssh_key_gen(self, env, ssh_user, is_customer_key=False):
 
         ext_project_ssh_priv_key = project_ssh_priv_key + '.pem'
         os.rename(project_ssh_priv_key, ext_project_ssh_priv_key)
-        if env.reference_architecture == 'EDB-Always-On':
+        if env.reference_architecture in ['EDB-Always-On-Platinum', 'EDB-Always-On-Silver']:
```

Comment:
Suggestion: maybe you could check if `env.reference_architecture` starts with `EDB-Always-On` like this:
```python
if env.reference_architecture.startswith('EDB-Always-On'):
```